modules/project_services/variables.tf

```diff
@@ -33,3 +33,10 @@ variable "disable_services_on_destroy" {
   default     = "true"
   type        = "string"
 }
+
+variable "disable_dependent_services" {
+  description = "Whether services that are enabled and which depend on this service should also be disabled when this service is destroyed. https://www.terraform.io/docs/providers/google/r/google_project_service.html#disable_dependent_services"
```

Comment:
very unclear description

Reply:
This description comes directly from the terraform documentation related to this 
argument,
https://www.terraform.io/docs/providers/google/r/google_project_service.html#disable_dependent_services

What exactly is not clear for you?
Thanks.